internal/model/service.go

```diff
@@ -4,20 +4,40 @@ import (
 	"context"
 	"fmt"
 	"strings"
+
+	"github.com/mitchellh/hashstructure"
 )
 
 type ServiceName string
 
 func (s ServiceName) String() string { return string(s) }
 
 type Service struct {
-	K8sYaml        string
+	K8sYaml        string `hash:"ignore"`
```

Comment:
Check this out @nicks. I didn't realize this was a feature of the library!

Reply:
:raised_hands: